apollo-router/src/services/external.rs

```diff
@@ -3,35 +3,27 @@
 
 use std::collections::HashMap;
 use std::fmt::Debug;
-use std::string::ToString;
 use std::time::Duration;
 
 use http::header::ACCEPT;
 use http::header::CONTENT_TYPE;
+use http::Method;
 use http::StatusCode;
-use once_cell::sync::Lazy;
-use reqwest::Client;
+use hyper::Body;
 use schemars::JsonSchema;
 use serde::de::DeserializeOwned;
 use serde::Deserialize;
 use serde::Serialize;
 use strum_macros::Display;
 use tower::BoxError;
+use tower::Service;
 
-use crate::error::LicenseError;
-use crate::services::apollo_graph_reference;
-use crate::tracer::TraceId;
 use crate::Context;
 
-const DEFAULT_EXTERNALIZATION_TIMEOUT: Duration = Duration::from_secs(1);
-
-static CLIENT: Lazy<Result<Client, BoxError>> = Lazy::new(|| {
-    apollo_graph_reference().ok_or(LicenseError::MissingGraphReference)?;
-    Ok(Client::new())
-});
```

Comment:
This is removing the requirement to have APOLLO_GRAPH_REF set to use this feature. At some point, this should be replaced with the proper entitlement support, but I'm not sure if we've reached that point yet.

Reply:
Right, the logic was moved [here](https://github.com/apollographql/router/blob/c8a82329ce57b5cc8c61277b60d8bb44eb292e75/apollo-router/src/plugins/external.rs#L56).